src/refinement-list/refinement-list.ts

```diff
@@ -70,17 +77,18 @@ export class NgAisRefinementList extends BaseWidget {
   // render options
   @Input() public showMoreLabel: string = 'Show more';
   @Input() public showLessLabel: string = 'Show less';
-  @Input() public transformItems?: Function;
   @Input() public searchable?: boolean;
   @Input() public searchPlaceholder: string = 'Search here...';
 
   // connectors options
   @Input() public attribute: string;
   @Input() public operator: 'or' | 'and' = 'or';
-  @Input() public limit: number | string = 10;
-  @Input() public showMoreLimit: number | string;
-  @Input() public showMore: boolean = false;
+  @Input() public limit: number = 10;
+  @Input() public showMore?: boolean;
+  @Input() public showMoreLimit: number;
```

Comment:
`showMoreLimit` is optional.

Reply:
you're right. 
I mistakenly thought angular used that prop to guide the user or error when required props are not provided (or type incompatible) but it doesn't seem so.
It seems the conventional way to  signal something is optional is by supplying a default value.

https://toddmotto.com/passing-data-angular-2-components-input#input-decorator-stateless-component

I'd prefer to use the default values provided by the connectors.
With no typing benefit the `?` as mostly overloading the code. I'm thinking of removing them altogether in component inputs. WDYT?

Reply:
We should indeed rely on the connectors' default values.

What do you mean by "removing them altogether in component inputs"? (what should be removed?)

I think the best way to make our minds would be to try that in the storybook.

Reply:
providing a default value everywhere might conflict with `includedAttributes` / `excludedAttributes`, since they aren't compatible at all and it will give a warning if both  are passed to the connector (even if they have a default there)